models/contact.settings.json

```diff
@@ -51,11 +51,9 @@
       "plugin": "crm-plugin",
       "model": "country"
     },
-    "villages": {
+    "village": {
```

Comment:
To generalise to meet the requirements for all existing projects we have added this relation "Village has many to many relation" . In order to access village for a specific contact incase of single village is to access the 0th element of the array.

Reply:
Alright that make sense to me. Anyway we won't be using villages in medha-STPC.